src/Zikula/FormExtensionBundle/Resources/views/Form/bootstrap_4_zikula_admin_layout.html.twig

```diff
@@ -158,7 +158,7 @@ col-md-9
     {% endif %}
     {# load required assets: jQuery UI + custom search code #}
     {{ pageAddAsset('stylesheet', asset('jquery-ui/themes/base/jquery-ui.min.css')) }}
-    {{ pageAddAsset('javascript', asset('jquery-ui/jquery-ui.min.js')) }}
+    {{ pageAddAsset('javascript', asset('jquery-ui/jquery-ui.min.js'), constant('Zikula\\ThemeModule\\Engine\\AssetBag::WEIGHT_JQUERY_UI')) }}
```

Comment:
Why are these needed? Shouldn't this be done automatically (in/by `validateBag`)?
Is this just for verbosity or is it required to add the weight explicitly for each `pageAddAsset` call now?

Reply:
it is not required (for jquery-ui only) of course, but frankly, I would prefer that it is set manually always and remove from validateBag entirely.

Reply:
> remove from validateBag entirely

please let's keep it until 4.0 when we finally switch to modern JS handling (ES6, webpack, etc.)